### PR TITLE
Add Xcake and Tuist as alternatives

### PR DESCRIPTION
**Pull request by natanrolnik**
Hello :)

This little change to the Readme adds two alternatives besides XcodeGen: Xcake (currently maintained by @igor-makarov) and Tuist (disclaimer: I'm a contributor).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,11 @@ See our [Contribution guidelines](./CONTRIBUTING.md).
 
 ## Alternatives
 
-- [XcodeGen](https://github.com/yonaskolb/XcodeGen): You can commit this JSON file into Git instead of the `.pbxproj` file. Then resolving conflicts is much easier.
+All of the alternatives below allow you to generate your Xcode projects based on a spec or manifest. You commit these files to git, and can even remove the `.xcodeproj` files from git. If you decide to keep them in git, resolving conflicts is much simpler, by generating the projects again.
```
Comment (byohay) [inline README.md:47]: I'd remove the last sentence, I prefer to have a general description as the readme's real estate is expensive.

**Review by byohay — CHANGES_REQUESTED** [review]
@natanrolnik, thanks for the PR!
Some small comments.

**Comment by byohay** [comment]
@natanrolnik please fix the comments or I'll close the PR :)

**Comment by natanrolnik** [comment]
@byohay done, sorry for dropping the ball on this.

**Review by byohay — APPROVED** [review]

**Pull request merged**